main.py

```diff
@@ -0,0 +1,117 @@
+from datetime import datetime
+"""
+open all 3 files
+parse such that we have the log in a list
+count how many lines are in log, level=error count, level=info
+
+TIME FORMAT : 08:56:35.354
+
+top 5 most requested HTTP paths/endpoints
+"""
+
+class Logging:
+    def __init__(self):
+        self.log = []
+        self.date_time = []
+
+
+    def logs(self, file1, file2, file3):
```

Comment:
Lastly on the topic of this function in general, If your class ever needs to do something before that class becomes useful it's probably best to put that into the `__init__` function.

So in this case, this class doesn't actually give the user any benefit until you have ingested the logs. So this function which is the log ingestion function should probably be the init function.